src/lib/accessibility/Behaviors/Radio/RadioGroupBehavior.ts

```diff
@@ -0,0 +1,29 @@
+import { Accessibility } from '../../interfaces'
+import * as keyboardKey from 'keyboard-key'
+
+/**
+ * @description
+ *  Adds role='radiogroup'. This allows screen readers to handle the component as a radio group.
+ *  Performs 'nextItem' action on ArrowDown, ArrowRight.
+ *  Performs 'prevItem' action on ArrowUp, ArrowLeft.
+ */
+const RadioGroupBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    root: {
+      role: 'radiogroup',
+    },
+  },
+
+  keyActions: {
+    root: {
+      nextItem: {
+        keyCombinations: [{ keyCode: keyboardKey.ArrowDown }, { keyCode: keyboardKey.ArrowRight }],
+      },
+      prevItem: {
+        keyCombinations: [{ keyCode: keyboardKey.ArrowUp }, { keyCode: keyboardKey.ArrowLeft }],
```

Comment:
Shouldn't we make navigation based on ```ArrowRight``` and ```ArrowLeft``` for horizontal radio group, and ```ArrowUp``` and ```ArrowDown``` for the vertical?

Reply:
according to ARIA spec, you can use both up/down and left/right to switch between the radio buttons.